packages/react-components/react-provider/src/components/FluentProvider/useFluentProviderStyles.ts

```diff
@@ -11,9 +11,7 @@ const useStyles = makeStyles({
   root: {
     color: tokens.colorNeutralForeground1,
     backgroundColor: tokens.colorNeutralBackground1,
-    fontFamily: tokens.fontFamilyBase,
-    fontSize: tokens.fontSizeBase300,
-    fontWeight: tokens.fontWeightRegular,
+    ...typographyStyles.body1,
```

Comment:
Leveraging `typographyStyles` to improve clarity on why we use said values and to avoid duplication.